galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/ras/directory/DirectoryRASDirectoryService.java

```diff
@@ -37,6 +38,30 @@ protected DirectoryRASDirectoryService(@NotNull Path baseDirectory, Gson gson) {
 		this.baseDirectory = baseDirectory;
 		this.gson = gson;
 	}
+	
+	@Override
+	public @NotNull List<IRunResult> getRuns(@NotNull IRasSearchCriteria... searchCriteria) throws ResultArchiveStoreException{
+		
+		ArrayList<IRunResult> runs = new ArrayList<>();
+		
+		List<DirectoryRASRunResult> allRuns = getAllRuns();
+		
+		boolean matched = Boolean.TRUE;
+	
+		for(DirectoryRASRunResult run : allRuns) {
+			for(IRasSearchCriteria criteria : searchCriteria) {
+				if(!criteria.criteriaMatched(run.getTestStructure())) {
+					matched = Boolean.FALSE;
```

Comment:
grrrr,   just false,    not Boolean.FALSE.    you are instantiating an object to set a primitive